installation/Makefile

```diff
@@ -2,28 +2,17 @@ API_DIR = ../estela-api
 QUEUING_DIR = ../queueing
 WEB_DIR = ../estela-web
 API_POD = $$(kubectl get pod -l app=estela-django-api -o jsonpath="{.items[0].metadata.name}")
-API_IP = $$(kubectl get services -n $${NAMESPACE} estela-django-api-service --output jsonpath='{.status.loadBalancer.ingress[0].ip}')
+API_IP = $$(kubectl get services -n $${NAMESPACE} estela-django-api-service --output jsonpath='{.spec.clusterIP}')
+LOCAL_API_IP = $$(kubectl get services -n $${NAMESPACE} estela-django-api-service --output jsonpath='{.status.loadBalancer.ingress[0].ip}')
 RESOURCES = db registry minio zookeeper kafka
 
 
-ifeq ($(OS),Windows_NT)
-	DOCKER_COMPOSE = docker compose
-else
-	UNAME_S := $(shell uname -s)
-	ifeq ($(UNAME_S),Linux)
-		DOCKER_COMPOSE = docker-compose
-	else # OSX
-		DOCKER_COMPOSE = docker compose
-	endif
-endif
-
```

Comment:
Why you delete this?

Reply:
@Adriana618 I removed this because `docker compose` command is officially supported on all platforms, making it no longer necessary to use `docker-compose` for Linux.